SampleScripts/ila/ilaDeployVrealizeLogInsight.ps1

```diff
@@ -60,7 +61,7 @@ Try {
             Write-LogMessage -type INFO -message "Opening the Excel Workbook: $Workbook"
             $pnpWorkbook = Open-ExcelPackage -Path $Workbook
             Write-LogMessage -type INFO -message "Checking Valid Planning and Prepatation Workbook Provided"
```

Comment:
Preparation

Reply:
Fixed with [39e6512](https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/pull/132/commits/39e65124ab99446ada010ec73a8a2b9ea0afd9e5)